packages/design-system-docs/src/pages/components/TextField/TextField.example.jsx

```diff
@@ -15,7 +15,7 @@ ReactDOM.render(
       label="Password field"
       name="password_example"
       type="password"
-      value="Example value"
+      defaultValue="Example value"
```

Comment:
The old password TextField example doesnt let you edit the text, this fixes that